docs/io/fits/usage/unfamiliar.rst

```diff
@@ -136,7 +137,7 @@ type specification (i.e. the value of the TFORM keyword) uses an extra letter
 
     rPt(max)
 
-where r is 0, 1, or absent, t is one of the letter code for regular table data
+where r is the column length (it can be absent), t is one of the letter code for regular table data
```

Comment:
In variable length array columns each element in the column is an array of zero or more elements.  The length of the array (in number of items of whatever scalar type) is given on a per-element basis in the array descriptors that are stored in the raw table data.  The contents of each element are then looked up in a "heap" that comes after the table data.  Hence there is no single "repeat count" that applies to every element in the column.

This is a relatively unusual data structure, and I think the general assumption of the docs here is that if you actually know you need to use this you will have some idea how the format works (as described in the FITS standard).